source/run-command/tests/unified/runCursorCommand.yml

```
@@ -0,0 +1,488 @@
+    expectEvents:
+      - client: *client
+        eventType: command
+        events:
+          - commandStartedEvent:
+              command:
+                drop: *cappedCollection
+              commandName: drop
+          - commandStartedEvent:
+              command:
+                create: *cappedCollection
+              commandName: create
+          - commandStartedEvent:
+              command:
+                insert: *cappedCollection
+              commandName: insert
+          - commandStartedEvent:
+              command:
+                find: *cappedCollection
+                filter: {}
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: find
+          - commandStartedEvent:
+              command:
+                getMore: { $$type: long }
+                collection: *cappedCollection
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: getMore
+          - commandStartedEvent:
+              command:
+                killCursors: *cappedCollection
+                cursors: [{ $$type: long }]
+              commandName: killCursors
+
+  # CSOT
+  - description: errors if timeoutMode is set without timeoutMS
+    operations:
+      - name: runCursorCommand
+        object: *db
+        arguments:
+          commandName: find
+          command: { find: *collection, filter: {} }
+          timeoutMode: cursorLifetime
+        expectError:
+          isClientError: true
+
+  - description: error if timeoutMode is cursor_lifetime and cursorType is tailableAwait
+    operations:
+      - name: runCursorCommand
+        object: *db
+        arguments:
+          commandName: find
+          command: { find: *collection, filter: {} }
+          timeoutMode: cursorLifetime
+          cursorType: tailableAwait
+        expectError:
+          isClientError: true
+
+  - description: error if getMore maxTimeMS setting is greater than timeoutMS
```

I don't recall any mention of this in the runCommand spec. It merely said that drivers SHOULD raise an error if both are specified.

If this is a CSOT requirement, I reckon these tests would be better placed under the CSOT spec itself.